index.html

```diff
@@ -4,44 +4,69 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  
     <title>Sabka Bazar</title>
-    <!-- <base href="/"> -->
+
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Dosis:400,600,700">
-    <link type="text/css" rel="stylesheet" href="./shared/css/style.css">
     <script src="http://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
-   
-    <script src="src/app/signup/signup.js"></script>
-    <script src="src/app/login/login.js"></script>
-    <script src="src/app/home/home.js"></script>
-    <script src="src/app/products/products.js"></script>
-    <script src="src/app/index.js"></script>
   </head>
   <body>
-    <header class="header">
+    <header>
       <nav class="navigation">
```

Comment:
If you are using HBS why this markup is part of your HTML page, it should be inside HBS components and at page (template) level you only import those components.

Reply:
Created a separate component for both Header and Footer.